prep-html-syntax/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+  <title>HTLM Syntax</title>
+  <style>
+    h1{color: goldenrod;}
+    p{color:silver;}
+  </style>
+</head>
+<h1> My favorite Star Wars character is c-3po.</h1>
+  <body>
+  <img src="https://m.media-amazon.com/images/I/71LFpKcZjGL._AC_SL1305_.jpg" alt="c-3po">
+  <p>C-3po is not my favorite, but he does come in clutch sometimes.</p>
+  </body>
+</html>
+</html>
```

Comment:
Hi Jesus, looks like the extra closing tag is still there. Be sure to save, add, and commit in that order so that Git can push your new commit to GitHub.